{
  setFailFast(false);
  parseExpression("try: 1 + 1");
  assertContainsError("'try' not supported, all exceptions are fatal");
  assertContainsError("syntax error at 'try': expected expression");
}
